homework/style.css

```diff
@@ -1,3 +1,112 @@
+* {
```

Comment:
Your styling is not up to par. This is what it looks like on a wide-screen monitor:

![image](https://user-images.githubusercontent.com/2788771/66043203-d1042480-e51e-11e9-9adf-25e9abff5573.png)

And on a mobile phone:

![image](https://user-images.githubusercontent.com/2788771/66043220-e11c0400-e51e-11e9-8117-833e4ba5a706.png)

Reply:
dear @remarcmij , i made the page responsive now,  it looks better,